Lib/test/test_email/test_message.py

```diff
@@ -486,11 +486,15 @@ def message_as_iter_attachment(self, body_parts, attachments, parts, msg):
         attachments = [allparts[n] for n in attachments]
         self.assertEqual(list(m.iter_attachments()), attachments)
 
+    def _is_multipart_msg(self, msg):
```

Comment:
This could be an inner function of message_as_iter_parts because it's only used there.